docs/changelog.rst

```diff
@@ -11,7 +11,22 @@ New Features
 Fixes
 -----
 
+Other Changes
+--------------
+
+0.4.4
+=====
+
+:Date: Apr 15, 2020
+
+New Features
+-------------
+
+Fixes
+-----
+
 * Fix definition lists looking different with Sphinx 2.0+
+* Add compatibility with Sphinx 3.x
```

Comment:
Where was Sphinx 3.x compatibility added?

Reply:
Commit a49a812c8821123091166fae1897d702cdc2d627

Reply:
3.0.0 was released a year later. That was for Sphinx 2.x compatibility:
https://github.com/sphinx-doc/sphinx/issues/6088

Reply:
Sphinx 2.x compatibility is still ongoing:
https://github.com/readthedocs/sphinx_rtd_theme/projects/1

Reply:
"Improve compatibility with Sphinx 2.x" is more appropriate